src/signals/incident-management/containers/Dashboard/RealtimeDashboard.tsx

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { StyledMenuInline, MenuChildren } from '@amsterdam/asc-ui'
+const RealtimeDashboard = () => {
+  return (
+    <>
+<StyledMenuInline>
```

Comment:
no need to wrap in <></>

Reply:
I've used an other component altogether.